src/main/resources/templates/in-app/sponsorship/v1.0.0.html

```diff
@@ -21,7 +21,12 @@ <h2>Content</h2>
         >.
       </p>
       <p>
-        If you require a Certificate of Sponsorship in order to apply for a Skilled Worker visa for this rotation, please follow this <a href="https://medical.hee.nhs.uk/medical-training-recruitment/medical-specialty-training/overseas-applicants/overseas-sponsorship-guidance/overseas-sponsorship-contact-details">link</a> for details on the application process. Should you have any questions that are not answered on this website,
+        If you require a Certificate of Sponsorship in order to apply for a Skilled Worker visa for this rotation, please follow this
+        <a
+          href="https://medical.hee.nhs.uk/medical-training-recruitment/medical-specialty-training/overseas-applicants/overseas-sponsorship-guidance/overseas-sponsorship-contact-details"
+          th:target="_blank"
+        >link</a>
```

Comment:
@DorisWongNHS , in true JFDI spirit, we could make this link text more descriptive e.g. instead of 'link' we could do 'link to the overseas sponsorship application process contact details'. It's still a bit wordy but at least the user should have more idea what they are clicking 😁 (ignore me if it's part of a follow-up ticket etc.)